src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -505,6 +505,96 @@ int F(int a)
                 Row(4, TableIndex.MethodDef, EditAndContinueOperation.Default));
         }
 
+        [Fact]
+        public void MethodWithNullable_AddingNullCheck()
+        {
+            var source0 = MarkedSource(@"
+using System;
+#nullable enable
+
+class C
+{
+    static T id<T>(T t) => t;
+    static T G<T>(Func<T> f) => f();
+
+    public void F(string? x)
+    <N:0>{</N:0>
+        var <N:1>y1</N:1> = new { A = id(x) };
+        var <N:2>y2</N:2> = G(<N:3>() => new { B = id(x) }</N:3>);
+        var <N:4>z</N:4> = G(<N:5>() => y1.A + y2.B</N:5>);
+    }
+}", options: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview));
```

Comment:
>options: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview) [](start = 4, length = 80)

Is this needed or is the default (C#8) sufficient?

Reply:
I don't think we should limit tests to specific version of the language. We always want to test the latest C#, in case there was some kind of regression.